devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -371,21 +371,67 @@ def getTemperature(value) {
 
 	def refresh() {
 		log.debug "Refreshing Values "
-		def refreshCmds = [
         
-        /* sensitivity - default value (8) */
+        def refreshCmds = []
         
-        "zcl mfg-code ${manufacturerCode}", "delay 200",
-        "zcl global write 0xFC02 0 0x20 {02}", "delay 200",
-        "send 0x${device.deviceNetworkId} 1 1", "delay 400",
+		if (device.getDataValue("manufacturer") == "SmartThings") {
```

Comment:
I think it would be preferable to just put the motion threshold related commands in an if/else block, since all the other commands are the same for both manufacturers. That way there is less duplicated code.